tasks.yaml

```diff
@@ -144,7 +146,7 @@ tasks:
     description: "Clone UDS Core for integration testing"
     actions:
       - cmd: rm -rf uds-core
-      - cmd: git clone --branch ${CORE_VERSION} https://github.com/defenseunicorns/uds-core.git
+      - cmd: git clone --branch 1630-ALB-Support https://github.com/defenseunicorns/uds-core.git
```

Comment:
Will want to revert when the core branch is merged.

Reply:
Switched to main for the time-being (will revert back after the release)